stablehlo/integrations/python/mlir/dialects/stablehlo.py

```diff
@@ -17,3 +17,11 @@
 # pylint: disable=wildcard-import,relative-beyond-top-level,g-import-not-at-top
 from ._stablehlo_ops_gen import *
 from .._mlir_libs._stablehlo import *
+
+
+def get_earliest_forward_compatible_version():
+  """Return the earliest StableHLO version that the current StableHLO version
+
```

Comment:
Looks like extraneous newline.

Reply:
removed method in favor of StablehloModule.cpp impl